test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -19,7 +19,7 @@
 TF_ADDONS_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "testTFAddons")
 TF_DATASERVICE_TEST_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "testDataservice")
 
-TF_EC2_SINGLE_GPU_INSTANCE_TYPE = get_ec2_instance_type(default="p2.xlarge", processor="gpu")
+TF_EC2_SINGLE_GPU_INSTANCE_TYPE = get_ec2_instance_type(default="p3.2xlarge", processor="gpu")
```

Comment:
There is also a telemetry test that uses p2.xlarge.

Reply:
Updated the instance type to p3.2xlarge